cmd/compile.go

```diff
@@ -47,16 +47,21 @@ a custom backend to target your own runtime.`,
 }
 
 func compileFunc(cmd *cobra.Command, args []string) {
-	if compileSettings.swaggerFile == "" {
+	if len(compileSettings.swaggerFiles) == 0 {
 		logrus.Fatal("swagger file is required for inferring types")
```

Comment:
Is it really fatal, maybe should be warning? Can't we compile without swagger files?

Reply:
I think it's impossible, because swagger files contains data that's used during compilation

Reply:
definitely fatal - types is required